packages/manager/apps/pci-public-ip/src/pages/list/List.page.tsx

```diff
@@ -71,97 +72,99 @@ export default function ListingPage(): JSX.Element {
   }, [projectId, navigation]);
 
   return (
-    <>
-      {project && (
-        <OsdsBreadcrumb
-          items={[
-            {
-              href: projectUrl,
-              label: project.description,
-            },
-            {
-              label: t('pci_additional_ips_title'),
-            },
-            {
-              label:
-                activeTab === IPsTabName.ADDITIONAL_IP_TAB_NAME
-                  ? t('pci_additional_ips_failover_ip_title')
-                  : t('pci_additional_ips_floating_ip_title'),
-            },
-          ]}
-        />
-      )}
-      <div className="header mb-10 mt-8">
-        <div className="flex items-center justify-between">
-          <OsdsText
-            level={ODS_THEME_TYPOGRAPHY_LEVEL.heading}
-            size={ODS_THEME_TYPOGRAPHY_SIZE._600}
-            color={ODS_THEME_COLOR_INTENT.primary}
-          >
-            {t('pci_additional_ips_title')}
-          </OsdsText>
-          <PciGuidesHeader category="instances"></PciGuidesHeader>
-        </div>
-        <div className="mt-4">
-          <OsdsText
-            color={ODS_THEME_COLOR_INTENT.text}
-            size={ODS_THEME_TYPOGRAPHY_SIZE._400}
-          >
-            {t('pci_additional_ips_additional_ips_description')}
-          </OsdsText>
-        </div>
-      </div>
-
-      {isBannerVisible && <PciAnnouncementBanner projectId={projectId} />}
-
-      <div className="mb-5">
-        {isDiscoveryProject(project) && (
-          <PciDiscoveryBanner projectId={projectId} />
+    <ListGuard projectId={projectId}>
+      <>
```

Comment:
nitpick, i think you can remove one nesting level :

```suggestion
    <ListGuard projectId={projectId}>
```

Reply:
the ListGuard receive a children as props and the children must be one wrapped element